src/infql.jl

```diff
@@ -378,3 +378,238 @@ function LazyBandedMatrices._SymTridiagonal(::Tuple{TriangularLayout{'L', 'N', P
     ev = [A[k,k+1] for k=1:m-1]
     SymTridiagonal([dv; Fill(dv[end],∞)], [ev; Fill(ev[end],∞)])
 end
+
+###
+# Experimental adaptive finite section QL
+###
+            
+mutable struct QLFiniteSectionTau{T} <: LazyArrays.AbstractCachedVector{T}
```

Comment:
Just because AbstractCachedVector isn't explicitly exported by LazyArrays. But it would be nicer to just import it into the package, I will change it.